web/js/naturalEvents/wv.naturalEvents.ui.js

```diff
@@ -9,6 +9,9 @@ wv.naturalEvents.ui = wv.naturalEvents.ui || function (models, ui, config, reque
   var naturalEventMarkers = wv.naturalEvents.markers(models, ui, config);
 
   var init = function () {
+    // Display loading information for user feedback on slow network
+    $('#wv-events').text('Loading...');
```

Comment:
This file has changed in the `module-loaders` branch. Let's try to change the target of this pull request to that branch, and manually adjust it if necessary.